src/current/v25.2/ldap-authentication.md

```diff
@@ -4,9 +4,13 @@ summary: Learn how to configure CockroachDB for user authentication using LDAP-c
 toc: true
```

Comment:
This page is broken on [preview](https://deploy-preview-20033--cockroachdb-docs.netlify.app/docs/v25.2/ldap-authentication). Not sure why but often has to do with a misformatted tag:

<img width="980" height="585" alt="image" src="https://github.com/user-attachments/assets/ddc3f01a-9fdb-4bfc-85ae-0d827523987a" />